libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -225,51 +286,71 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
     }
   }
 
-  private checkForEmbeddedFooter() {
-    const embeddedComponentElement = this.ionContentElement.nativeElement.firstElementChild;
-    if (embeddedComponentElement) {
-      this.embeddedFooterElement = embeddedComponentElement.querySelector('kirby-modal-footer');
-      this.moveEmbeddedFooter();
-      this.observeEmbeddedFooter(embeddedComponentElement);
-    }
+  private readonly elementToParentMap: { [key: string]: () => HTMLElement } = {
+    'KIRBY-MODAL-FOOTER': () => this.elementRef.nativeElement,
```

Comment:
Why are these wrapped inside an arrow-func? :)

Reply:
Because `this.elementRef.nativeElement` is not present at declaration time :)